apps/src/templates/teacherDashboard/EditSectionForm.jsx

```diff
@@ -130,6 +139,49 @@ class EditSectionForm extends Component {
       isNewSection
     } = this.props;
 
+    /**
+    OAuth login types can not be changed.
+    Personal email login type can be changed to word OR picture login type.
+    Picture login type can be changed to word login type.
+    Word login type can be changed to picture login type.
+    **/
+    const changableLoginTypes = [
+      SectionLoginType.word,
+      SectionLoginType.picture,
+      SectionLoginType.email
+    ];
+
+    // Sections that do not yet have students can use any login type.
+    var validLoginTypes;
+    switch (true) {
+      case section.studentCount === 0:
+        validLoginTypes = [
+          SectionLoginType.word,
+          SectionLoginType.picture,
+          SectionLoginType.email,
+          SectionLoginType.google_classroom,
+          SectionLoginType.clever
+        ];
+        break;
+      case section.loginType === SectionLoginType.email:
+        validLoginTypes = changableLoginTypes;
+        break;
+      case section.loginType === SectionLoginType.picture ||
+        section.loginType === SectionLoginType.word:
+        validLoginTypes = [SectionLoginType.word, SectionLoginType.picture];
+        break;
+      case section.loginType === SectionLoginType.clever:
```

Comment:
We shouldn't ever see dropdown for Google or Clever sections, but just in case...